source/backend/core/Pims.Core.csproj

```diff
@@ -10,6 +10,7 @@
   <ItemGroup>
     <PackageReference Include="ClosedXML" Version="0.95.4" />
     <PackageReference Include="CsvHelper" Version="27.1.0" />
+    <PackageReference Include="Macross.Json.Extensions" Version="2.2.0" />
```

Comment:
Do we need this extension for the JsonFilter to work? I don't see it referenced anywhere. Perhaps a left-over of previous work?

Reply:
I'll double check, I originally added this for the new filter attribute but have since made changes that may no longer require this to be necessary.